thoth/common/helpers.py

```diff
@@ -169,3 +169,8 @@ def __get__(self, obj: object, _: Any = None) -> Any:
         value = self._calculate(obj)
         setattr(obj, self._calculate.__name__, value)
         return value
+
+
+def get_justification_link(identifier: str) -> str:
+    """Construct a link to a detailed justification document."""
+    return f"https://thoth-station.ninja/justifications/{identifier}.html"
```

Comment:
don't we want this to be an env variable as maybe we are going to change something in the future?

Reply:
Good idea, done in f4287ea